### PR TITLE
theme color and text theme

### PR DESCRIPTION
**Pull request by asoi1990**
Implement theme for color and text.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,70 @@ android/app/google-services.json
 ios/firebase_app_id_file.json
 ios/Runner/GoogleService-Info.plist
 lib/firebase_options.dart
+macos/firebase_app_id_file.json
```
Comment (hellomentorplus) [inline .gitignore:61]: u can just add 1 line: macos/**
coverage/**

**Comment by hellomentorplus** [comment]
where is your font family?

File: lib/main.dart [hunk 2]
```diff
@@ -30,9 +31,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
```
Comment (hellomentorplus) [inline lib/main.dart:32]: move MyApp to new file, and add main.dart into codecov ignore

File: test/theme/hs_text_theme_test.dart [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+  testWidgets('Validate [CAPTION] text style ', (WidgetTester tester) async {
+    GlobalKey smallK = GlobalKey();
+
+    final Builder displayTextGroup = Builder(builder: (BuildContext context) {
+      return SingleChildScrollView(
+        child: Column(
+          children: <Widget>[
+            Text('Test Small',
+                key: smallK, style: Theme.of(context).textTheme.bodySmall),
```
Comment (hellomentorplus) [inline test/theme/hs_text_theme_test.dart:160]: Column is sad, he wants multiple children, but he has only 1 single child

**Review by hellomentorplus — COMMENTED** [review]
good job

**Review by suesitran — APPROVED** [review]

**Pull request merged**